docs/introduction/faq.md

```diff
@@ -0,0 +1,95 @@
+
+The Interchain Foundation will suggest that 5% of the Atoms go to its initial donors, 10% go to the Interchain Foundation, 10% go to the company developing most of the software, and the remaining 75% to be distributed according to the results of the private and public fundraisers.
+
+### What is the team developing the Cosmos Network?
+
+The Cosmos Network is the first project being funded by the Interchain Foundation. Its development is led primarily by the [Tendermint team](https://tendermint.com/about).
+
+### What's the difference between Tendermint, the Cosmos Network, and the Cosmos Hub?
+
+* [Tendermint](https://tendermint.com) is a general purpose blockchain engine that uses a Byzantine-fault tolerant consensus protocol and allows applications to be written in any programming language.
+* The Cosmos Network is a heterogenous network of Proof-of-Stake blockchains that can interoperate with one-another.
+* The Cosmos Hub is the first Proof-of-Stake blockchain to be launched by the Cosmos Network; it uses Tendermint consensus, contains a built in governance protocol, and serves as co-ordinater for interoperability between other blockchains.
+* Atoms are the native cryptocurrency of the Cosmos Hub. Atoms are necessary for participating in the consensus protocol and transacting on the network.
+
+### When will the Cosmos Network launch?
+
+Please check [our roadmap](https://cosmos.network/roadmap) for the list of milestones wore aiming to achieve in order to launch.
+
+### What is the utility of Atoms?
+
+Public, decentralized networks require high levels of security and spam-prevention that are best achieved by economic means: participants in the consensus must incur some economic cost, and all transactions processed by the network must pay a fee. Since we want to use [Proof-of-Stake validators](/validators/overview.md) instead of Proof-of-Work miners, we require validators of the Cosmos Hub to make a large security deposit in Atoms - if they misbehave, their Atoms are revoked by the protocol! 
+
+The more Atoms in security deposits, the more stake on the line; the more skin-in-the-game; the greater the economic security. In this sense, the Atoms act like virtual miners.
+
+To achieve spam-prevention, all transactions on the Cosmos Hub must pay a fee in Atoms. The fee may be proportional to the amount of computation required by the transaction, similar to Ethereum's concept of "gas". The fees are collected by the validators and distributed proportionately to the Atoms held in security deposits.
```

Comment:
I think we should remove the last sentence here if we go with the changes above ☝️